<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>230</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>230</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>230</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>10</y>
     <width>391</width>
     <height>210</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Program mode:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="onlineButton">
        <property name="text">
         <string>Online</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="offlineButton">
        <property name="text">
         <string>Offline</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_1">
      <property name="text">
       <string>File name (only for offline mode):</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="filePathLine"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>latitude:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="latitudeLine"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>longitude:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="longitudeLine"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>minT:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="minTLine"/>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>maxT:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="maxTLine"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="startButton">
      <property name="text">
       <string>Start!</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
